data/brands/shop/furniture.json

```diff
@@ -436,7 +436,22 @@
       }
     },
     {
-      "displayName": "Habitat",
+      "displayName": "Habitat (UK)",
+      "locationSet": {
+        "include": [
+          "gb"
+        ]
+      },
+      "tags": {
+        "brand": "Habitat",
+        "brand:wikidata": "Q689059",
+        "brand:wikipedia": "en:Habitat_(retailer)",
+        "name": "Habitat",
+        "shop": "furniture"
+      }
+    },
+    {
+      "displayName": "Groupe Habitat",
```

Comment:
I agree with you and with the modification suggested by @bhousel .

Just be careful that interior_decoration.json aslo has a Habitat (UK).
In France at least, the Groupe Habitat sells both furnitures and interior decorations but I believe they focus a bit more on furnitures + Almost all the Habitat shops in France are mapped with shop=furniture. I'm not sure about the british branch though. What do you think ?

Reply:
Good spot @nuxper . That's got yet another wikidata ID!:
https://github.com/osmlab/name-suggestion-index/blob/ea9d441ffb3e1594fca160910219929b4393eb12/data/brands/shop/interior_decoration.json#L76-L89

The UK seems to have a mix of homewares/interior decoration/furniture currently!

The UK site lists stuff that's not furniture too, so maybe interior decoration is better.

@bhousel should it fail if there are two instances of a place with overlapping match tags (to catch issues like this)?

Reply:
@peternewman currently it will only fail if a duplicate id is generated.  The ids are generated by name + a hashcode that is unique across tree, key, value, locationset

I don't think the build script will fail if there is an overlapping matchtag, so that's a good suggestion..

Reply:
> @peternewman currently it will only fail if a duplicate id is generated. The ids are generated by name + a hashcode that is unique across tree, key, value, locationset

Is tree the file path?

> I don't think the build script will fail if there is an overlapping matchtag, so that's a good suggestion..

I've created #4921 to track this. It certainly hasn't complained currently.

Reply:
I haven't looked at this lately, but is is ok to merge? 
The decision is really whether we want to nudge all of the Habitat stores to use a consistent tag or whether we allow mappers to choose between furniture and interior_decoration.